scripts/pre_commit_linter.py

```diff
@@ -1019,24 +1027,29 @@ def _check_import_order(all_files):
         any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
         and filename.endswith('.py')]
     failed = False
-    for filename in files_to_check:
-        # This line prints the error message along with file path
-        # and returns True if it finds an error else returns False
-        # If check is set to True, isort simply checks the file and
-        # if check is set to False, it autocorrects import-order errors.
-        if (isort.SortImports(
-                filename, check=True, show_diff=True).incorrectly_sorted):
-            failed = True
+    with _redirect_stdout(_TARGET_STDOUT):
+        for filename in files_to_check:
+            # This line prints the error message along with file path
+            # and returns True if it finds an error else returns False
+            # If check is set to True, isort simply checks the file and
+            # if check is set to False, it autocorrects import-order errors.
+            if (isort.SortImports(
+                    filename, check=True, show_diff=True).incorrectly_sorted):
+                failed = True
+                print ''
+
     print ''
     print '----------------------------------------'
     print ''
     if failed:
         summary_message = (
             '%s   Import order checks failed' % _MESSAGE_TYPE_FAILED)
+        print summary_message
```

Comment:
Ditto, also, do away with the repetition.

Reply:
I think there was supposed to be a print statement here to print the summary message like every other function in `pre_commit_linter` has. It is added to print the summary_message -- it is in sync with the other functions

Reply:
Repetition? Can you pull out `print summary_message`?

Reply:
Um can you point out where is the existing `print summary_message` so I can put away with the repetition? As far as I know there is not a `print summary_message` in this function (`_check_import_order`). Thanks